### ISSUE TITLE
Refresh home page

### ISSUE THREAD

**Issue by philbarker** [t3-innovation-network/desm#600]
Refresh home page to avoid showing redundant and duplicate buttons and to improve clarity of what each button does.

This task is associated with the  General UX/UI Updates, issue #588  

The current home page for someone who is not logged in show:
 
![image](https://github.com/user-attachments/assets/dd459a0c-0d16-4472-a5ec-0d98259cbe04)

There is a lot of variation in how the buttons are shown: why are some on the top and others on the side? why are some white-on-black others white?  I don't think this helps.

The only button that would actually work are "View Shared Mappings" and "Sign In".  Please show only this button as active at this stage. Other buttons may be shown if greyed out. 

"View Shared Mappings" should be renamed as "View Published Mappings" or maybe just "Published Mappings"

When I log in as a standard user, I see

![image](https://github.com/user-attachments/assets/bc44db42-3abd-4b66-8cef-79ddf88dd983)

It is not 100% obvious that the user icon and text,  ![image](https://github.com/user-attachments/assets/d8fe06bc-2e32-467d-9de0-a48a8b0219ba) act as a link to user information.  Maybe use the Sign In / Sign Out button for this and make sign out and option in a drop-down that allow options such as: edit profile info, change password, choose current project.

What does "View Specification To see crosswalks currently in process." relate to? It is above the button that provides access to the Published Mappings not the work in progress.

The "View Shared Mappings" needs to be renamed, as "View Work In Progress" or just "Work in Progress" or "Your Mappings", to distinguish it from "View Mappings".

(see also issue #560)

I think we only need one New Mapping button.

Personally, I think the information under "Map Your Schema" should be on the page that you get to after hitting the "New Mapping" button, not here.

When I sign in as an Admin User, the only change is in the right of the top bar, where I get a button to link to the dashboard 
![image](https://github.com/user-attachments/assets/8b2501f3-5ec6-4b9f-a092-ce01e81acf3a)

I think it would be better not to spread the button that link to different activities all over the interface like this.  I always have to look for this button, and access to it is the only reason why I would log in as admin. Maybe put this with the other buttons.

As an Admin user I cannot access the View Mapping or New Mapping functions, so either don't show these buttons or grey them out.

**Comment by jeannekitchens** [comment]
@philbarker I agree with the issues you identified and they seem easy to fix.  Also, the landing page text is ok...though is it aligned to how we advertise and describe it and I have never seen the graphic used anywhere else and really don't feel it's helpful.  I'd rather see the flyers linked that were created this year and graphics that we've adopted and used in the flyers. 

Flyers are linked below.  The updated instructions are also relevant that are on the github site.

https://www.t3networkhub.org/resources/guidelines-for-planning-a-successful-data-ecosystem-schema-desm-mapping-project

https://www.t3networkhub.org/resources/data-ecosystem-schema-mapping-tool-desm

**Comment by dlawsonQIP** [comment]
The screen below is what you see when you first log in before clicking on anything.  I am logged in as  Deniselawson + CEDS@qi-partners.com

Note: Education & Workforce Schema Mapping is an available option for configuration profile.

![Image](https://github.com/user-attachments/assets/87ba0d60-625b-4d6e-a091-4fada676f5f2)


From the first screen below, when you select, **View Shared Mappings**, the dropdown list for choose configuration…. is different than what you see when you login initially (the screen above).  

![Image](https://github.com/user-attachments/assets/c5198a91-225f-4c76-955e-32a52deb6b01)

Below shows the drop-down list for configuration profile  (after clicking on View Shared Mappings) At this point, Education & Workforce Schema Mapping is not part of the drop down list of configuration profiles. Although this might make sense as the mapping hasn’t been created yet, it may cause confusion for some. 

![Image](https://github.com/user-attachments/assets/9ce4761b-af56-4865-9d76-8fc2ad713bd6)

Using the terminology of configuration profile is likely to be confusing.  Consider using different terminology such as "Mapping Projects" for the drop-down list.   View Shared Mappings vs. View Mappings also may cause confusion for users as the difference between what is displayed for each option is not clear to the user.  (View Shared Mappings doesn't require a login, it shows all published mappings. The list doesn't change when you are logged in.   View mappings requires a login and shows the mapping projects that the user is attached to.)

Changing the names of these options could provide more clarity as to which mappings will be listed.   I think it is important to make this distinction between "Published Mappings" that are available vs.  "My Mappings" from the user perspective.

**Comment by philbarker** [comment]
**Resolution**

- [ ] Copy the issues regarding UI terminology on the home page to #624 
- [ ] The text in the doc [provided by Denise](https://github.com/t3-innovation-network/desm/issues/600#issuecomment-3058842861) should be the default text for home pages of DESM instances.
- [ ] Owners of any instance can change the text, as it is in a separate file, but please add information on how to do this to the documentation on how to set up an instance (currently the github project readme).

**Comment by philbarker** [comment]
Looks good on gopher, moving to QA passed.
